config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,8 @@
+Sidekiq.configure_server do |config|
```

Comment:
It's exciting to use redis, glad we get to introduce our team to this concept and the powerful technologies surrounding them.